npm/webpack-dev-server/src/startServer.ts

```diff
@@ -21,5 +27,5 @@ export async function start (userWebpackConfig = {}, { specs, config, devserverE
 
   debug('starting webpack dev server')
 
-  return new WebpackDevServer(compiler, { hot: true, noInfo: !debug.enabled })
+  return new WebpackDevServer(compiler, { hot: true })
```

Comment:
Now that we have to wait for the dev server to finish compiling (due to the race condition bug), seeing the dev server is very important